iroh/src/magicsock.rs

```diff
@@ -1587,6 +1594,7 @@ impl Handle {
             return;
         }
         self.msock.closing.store(true, Ordering::Relaxed);
+        self.msock.cancel_transports.cancel();
```

Comment:
Instead of a new token, would this be done better as a child token from the `actor_token` that already exists on the line below?